src/handbook/company/achieving-success.md

```diff
@@ -0,0 +1,64 @@
+---
+navTitle: KPIs and OKRs
+meta:
+    title: KPIs and OKRs
+---
+
+# Achieving Success at FlowForge
+
+There's multiple ways to align a company around business goals, and at FlowForge
+we've identified two ways to help us shape this. The core metric are captured in
+the [KPI](#kpi). A KPI is a tool to identify problems and areas for improvement.
+
+The second tool employed at FlowForge to improve on the identfied problems are [OKR](#okr)s.
+
+## KPI
+
+A KPI, or Key Point Indicator, is used as a heuristic metric for company wide
+success. At FlowForge we're engaged in many initiatives to grow our offering and
+assess product market fit. At this time there's one KPI to track progress.
+
+### Managed Node-RED instances
+
+[Dashboard](https://datastudio.google.com/reporting/90315614-39bf-4eac-ba3d-45b4c635940e/page/P1V6C)
+
+FlowForge is the key product we're verifying in the market currently. Each
+customer electing to use FlowForge to help them in their Node-RED journey is a
+signal we're providing value.
+
+An instance is either a remote Node-RED running the [FlowForge Device Agent](https://flowforge.com/docs/user/devices/),
+a hosted Node-RED in the FlowForge Cluster, or an otherwise FlowForge licensed 
+and supported Node-RED instance.
+
+Telemetry collection is opt-in, and dependant on an internet connection. As such
+it's anticipated to under report the true number of FlowForge managed instances.
+
+Target growth for this metric is 10% week over week.
+
+## OKRs
+
+OKRs, Objectives and Key Results, are action oriented goals that have a _measureable_
+impact on the company. OKRs are intended to be bold goals, where the bar is set
+high enough that one still takes a leap.
+
+Given goals are intended to be bold and aggressive, performance on the OKRs has
+no influence on compensation or any other performance indicator of a department
+or individual.
+
+### Anatomy of an OKR
+
+Per timeboxed period a couple of OKRs are created. Each of them should describe
+clearly what the objective is, the plan to achieve that goal, and how success is
+measured. Further, the goal and plan to achieve it should align with the
+[values](./values/).
+
+### The process
+
+Before the OKR period, the CEO sets company wide OKRs. Each of the CEO, CTO,
+Head of Marketing, Head of UX, and product manager are expected to read the OKRs
+and create 2 OKRs of their own to help to achieve the company wide goal.
```

Comment:
```suggestion
and create 2 OKRs of their own to help to achieve the company wide goal.

A retrospective on the OKRs will be held at the end of the OKR timespan. This will ensure we all
learn about what works and what doesn't for the company.
```